package.json

```diff
@@ -20,7 +20,6 @@
     "dev": "electron-webpack dev",
     "compile": "electron-webpack && yarn extract-langs",
     "build": "yarn compile && electron-builder build",
-    "build:dir": "yarn build -- --dir -c.compression=store -c.mac.identity=null",
```

Comment:
We should keep this. It was requested here https://github.com/lbryio/lbry-app/pull/933.

Reply:
Just a merge error; will fix.